File: Exercism/c/resistor-color/src/resistor_color.c
#include "resistor_color.h"


uint16_t colorCode(resistor_band_t color) {
  return color;
}

const int *colors() {
  static const int all_colors[] = {
    BLACK, BROWN, RED, ORANGE, YELLOW, GREEN, BLUE, VIOLET, GREY, WHITE
  };
  return all_colors;
}

File: Exercism/javascript/resistor-color/resistor-color.js
export const colorCode = (color) => {
    let value = COLORS.indexOf(color.toLowerCase());
    return value >= 0 ? value : null;
};

export const COLORS = [
    "black"
    , "brown"
    , "red"
    , "orange"
    , "yellow"
    , "green"
    , "blue"
    , "violet"
    , "grey"
    , "white"
];
